certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/message/model/Answer.java

```diff
@@ -0,0 +1,24 @@
+package se.inera.intyg.certificateservice.domain.message.model;
+
+import java.time.LocalDateTime;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@Getter
```

Comment:
In a sense we control mutability as it is, because it only has getters and no setters. I think these classes are domain entities that can change (they have status for example), so we probably will have to change them in controlled ways as we do with Certificate and later save updates through the repository. What do you think about keeping them as is and the continued implementation of messages will decide?

Reply:
Sounds good to me!